language/vm/src/file_format.rs

```diff
@@ -1217,72 +1227,32 @@ pub struct CompiledScript(CompiledScriptMut);
 /// A mutable version of `CompiledScript`. Converting to a `CompiledScript` requires this to pass
 /// the bounds checker.
 #[derive(Clone, Default, Eq, PartialEq, Debug)]
-#[cfg_attr(any(test, feature = "testing"), derive(Arbitrary))]
-#[cfg_attr(any(test, feature = "testing"), proptest(params = "usize"))]
 pub struct CompiledScriptMut {
     /// Handles to all modules referenced.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any::<ModuleHandle>(), 0..=params)")
-    )]
     pub module_handles: Vec<ModuleHandle>,
     /// Handles to external/imported types.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any::<StructHandle>(), 0..=params)")
-    )]
     pub struct_handles: Vec<StructHandle>,
     /// Handles to external/imported functions.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any::<FunctionHandle>(), 0..=params)")
-    )]
     pub function_handles: Vec<FunctionHandle>,
 
     /// Type pool. All external types referenced by the transaction.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any::<TypeSignature>(), 0..=params)")
-    )]
     pub type_signatures: TypeSignaturePool,
     /// Function signature pool. The signatures of the function referenced by the transaction.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any_with::<FunctionSignature>(params), 0..=params)")
-    )]
     pub function_signatures: FunctionSignaturePool,
     /// Locals signature pool. The signature of the locals in `main`.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(any_with::<LocalsSignature>(params), 0..=params)")
-    )]
     pub locals_signatures: LocalsSignaturePool,
 
-    /// String pool. All literals and identifiers used in this transaction.
-    #[cfg_attr(
-        any(test, feature = "testing"),
-        proptest(strategy = "vec(\".*\", 0..=params)")
-    )]
+    /// String pool. All identifiers used in this transaction.
     pub string_pool: StringPool,
```

Comment:
Shall we rename the old `string_pool` to be identifier pool?

Reply:
Going to do that in another PR :)